api/apps/geoprocessing/test/steps/given-scenario-pu-data-exists.ts

```diff
@@ -187,3 +188,40 @@ export const GivenScenarioPuData = async (
     };
   });
 };
+
+export const GivenProjectPuData = async (
```

Comment:
have you checked if we don't have these elsewhere? looking at what these fixtures do rings some kind of bells... but I'm not sure :/

Reply:
I didn't find anything equivalent at first glance, but I did notice there was some redundancy to the way I refactored the functions so that it would affect other tests suites that were already using the combined scenario + project Pu function. Now it should look better